tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -171,3 +171,9 @@ jobs:
       searchFolder: '$(Build.BinariesDirectory)'
       testRunTitle: 'Unit Test Run'
     condition: succeededOrFailed()
+
+  - script: |
```

Comment:
But we have a "Clean Agent Directories" task. It doesn't work?

Reply:
> But we have a "Clean Agent Directories" task. It doesn't work?
@snnn This PR is obsolete
It's solved by the PR https://github.com/microsoft/onnxruntime/pull/14120